packages/eslint-plugin/tests/rules/no-identifier-enum-member.test.ts

```diff
@@ -0,0 +1,44 @@
+import rule, { MessageId } from '../../src/rules/no-identifier-enum-member';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('prefer-literal-enum-member', rule, {
+  valid: [
+    `
+enum Valid {
+  A = /test/,
+  B = 'test',
+  C = 42,
+  D = null,
+  E,
+  F = {},
+  G = [],
+  H = new Set(),
+  I = 2 + 2,
+}
+    `,
```

Comment:
please split these into separate cases.
If there is a regression for some reason, it becomes hard to debug this test.

Reply:
I can do this, but I was copying the pattern from other tests I found.  Also, the error does give the line number which caused the failure, so tracking down an issue shouldn't be a problem.

Reply:
> the error does give the line number

It does, but it's hard to decipher. You have to look up which test it is, then parse the json blob error to find the line number, then look up the line.

If it were separate tests, you would just need to check the failing code to find the exact failing case.